ansible_collections/arista/avd/roles/eos_designs/templates/designs/l3ls-evpn/tenants/vlan-interfaces.j2

```diff
@@ -13,6 +13,8 @@ vlan_interfaces:
                               svi_profile.tags) %}
 {%             set svi_name = svi_config.name | arista.avd.default(
                               svi_profile.name) %}
+{%             set svi_description = svi_config.description | arista.avd.default(
+                              svi_name) %}
```

Comment:
Missing extra whitespaces

Reply:
I can't see where the extra spaces go. Lines 16 and 17 seem to be correctly aligned with 14 and 15?

Reply:
Fixed extra white spaces :)